README.md

```diff
@@ -51,7 +51,17 @@ Contains the JSON schemas used to validate the sources in the integration tests.
 
 ## GTFS Realtime Data Structure
 
-Still in progress. [Please review the working document to provide your feedback](https://docs.google.com/document/d/1Mlz3AXHItInitsOEAKKi8hZV9d3t3gpFbyIZGm67ESU/edit#heading=h.88f252jw0urf).
+|     Field Name     |  Required from users  |                                                                              Definition                                                                             |
+|:------------------:|:---------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------:
+| MDB Source ID      | No - system generated | Unique numerical identifier.      |   |   |
+| Data Type          | Yes                   | The data format that the source uses: GTFS.                                                                                                            |   |   |
+| Provider           | Yes                   | Name of the transit provider.                                                                                                                                       |   |   |
+| Name               | Optional              | An optional description of the data source, e.g to specify if the data source is an aggregate of multiple providers, or which network is represented by the source. |   |   |
+| Static Reference | Optional                   | The MDB ID of the GTFS Schedule source associated with the Realtime source. If this is not provided, the country code, subdivision, municipality and bounding box will display as "Unknown" in the CSV export. If it is provided, these CSV fields will match the information in the static reference.                                                                                                                  |   |   |
```

Comment:
The current behavior is that country code, subdivision and municipality are set to unknown, and the bounding box coordinates are blank. I think we would need to update the description here, or adjust to behavior to it.